packages-native/safe-area-view/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "safe-area-view",
     "widgetName": "SafeAreaView",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
Isn’t better to just patch to 1.0.1 as we don’t have any new feature implemented?